tonic/src/transport/channel/service/tls.rs

```diff
@@ -26,12 +26,14 @@ pub(crate) struct TlsConnector {
 }
 
 impl TlsConnector {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
That's a bit cheating. If you want, I can refactor this, either using a parameter structure like

```rust
TlsConnectorParameters<'a> {
    ca_certs: Vec<Certificate>,
    trust_anchors: Vec<TrustAnchor<'static>>,
    identity: Option<Identity>,
    domain: &'a str,
    assume_http2: bool,
    use_key_log: bool,
}
```

or a builder pattern.

Reply:
this is fine, this is a private fn